hooks/schema-generation.js

```diff
@@ -0,0 +1,41 @@
+const {TypeScriptGenerator, FormatHelpers} = require('@asyncapi/generator-model-sdk');
+const fs = require('fs');
+const Path = require('path');
+
+const preset = {
+  interface: {
+    async property({ renderer, propertyName, property }) {
+      const name = propertyName;
+      const signature = renderer.renderTypeSignature(property, { isRequired: renderer.model.isRequired(propertyName) });
+      return `${name}${signature};`;
+
+    }
```

Comment:
```suggestion
    async property({ renderer, propertyName, property }) {
      const signature = renderer.renderTypeSignature(property, { isRequired: renderer.model.isRequired(propertyName) });
      return `${propertyName}${signature};`;

    }
```

Reply:
Why you want to use `renderer.model` instead of `model`?

```ts
    async property({ renderer, model, propertyName, property }) {
      const signature = renderer.renderTypeSignature(property, { isRequired: model.isRequired(propertyName) });
      return `${propertyName}${signature};`;
    }
```

Reply:
Didnt think `model` was being passed on to the arguments. At least I could not find any information about it.
```suggestion
    async property({ renderer, model, propertyName, property }) {
      const signature = renderer.renderTypeSignature(property, { isRequired: model.isRequired(propertyName) });
      return `${propertyName}${signature};`;
    }
```

Reply:
Gonna change it then.

Reply:
:P 

![image](https://user-images.githubusercontent.com/20404945/109297785-d7d9b880-7832-11eb-9e47-09dd5eccc97c.png)

Reply:
Apparently I did not read that close enough, thanks for the heads up 😄

Reply:
Don't worry, I also make tl;dr; hah 😃